com.unity.mobile.notifications/Runtime/iOS/iOSNotificationAttachment.cs

```diff
@@ -0,0 +1,20 @@
+namespace Unity.Notifications.iOS
+{
+    /// <summary>
+    /// Notification attachment.
+    /// Refer to Apple documentation for details.
+    /// </summary>
+    /// <see cref="https://developer.apple.com/documentation/usernotifications/unnotificationattachment?language=objc"/>
+    public struct iOSNotificationAttachment
```

Comment:
If it's not allowed to change Id/Url after the creation of iOSNotificationAttachment, consider doing this:

```
  public struct iOSNotificationAttachment
  {
        /// <summary>
        /// A unique identifier for the attachments. Will be auto-generated if left empty.
        /// </summary>
        public string Id { get; }

        /// <summary>
        /// URL to local file, accessible to the application.
        /// </summary>
        public string Url { get; }
       public iOSNotificationAttachment(string id, string url)
       {
           Id = id;
           Url = url;
       }
    }
```

Reply:
It's fine to change them.